plat/common/x86/cpu_native.c

```diff
@@ -55,13 +55,21 @@ unsigned long read_cr2(void)
 /* This corresponds to an 85 (42 << 1 | 1) return value from QEMU */
 #define QEMU_ISA_DEBUG_EXIT_CRASH	42
 
+/* The port used by QEMU by default for the pvpanic device */
+#define QEMU_PVPANIC_EXIT_PORT		0x505
+/* This corresponds to a GUEST_PANICKED (1 << 0) event for QEMU */
+#define QEMU_PVPANIC_GUEST_PANICKED	1
+/* This corresponds to a GUEST_CRASHLOADED (1 << 1) event for QEMU */
+#define QEMU_PVPANIC_GUEST_CRASHLOADED	2
+
 /**
  * Trigger an exit() in QEMU with the code `value << 1 | 1`.
  * @param value the value used in the calculation of the exit code
  */
 static void qemu_debug_exit(int value)
 {
 	outw(QEMU_ISA_DEBUG_EXIT_PORT, value);
+	outw(QEMU_PVPANIC_EXIT_PORT, QEMU_PVPANIC_GUEST_PANICKED);
```

Comment:
That breaks the "normal" exit code (see line 109). And it does not fit into this function, because this function is for the isa-debug-exit device.

Reply:
Sure, I can just create another function

Reply:
actually, it's a single place where that is used, I just moved it down

Also changed to `outb` cause it's a single byte